modules/common/src/test/java/com/ibm/cloud/cloudant/SdkCommonTest.java

```diff
@@ -37,7 +37,7 @@ public void testGetSdkHeaders() {
         assertTrue(headers.containsKey("User-Agent"));
```

Comment:
Can you update the copyright header to `© Copyright IBM Corporation 2020, 2022. All Rights Reserved.`?

Reply:
Now that we've gone back to the format with spaces I've reverted this file so this isn't necessary

Reply:
Now that we've gone back to the format with spaces I've reverted this file so this isn't necessary